### PR TITLE
Add Thermal NVGs

### PR DESCRIPTION
**Pull request by OriUnder**
This adds new NVG item that adds 3 form factors with White Hot and Orange Hot thermals with potential expansion for snow (or other) textures

Gungnir
Marksman NVG
NVG Thermal Meta Item

### CODE REVIEW

File: PF_Gear/nvgs/nvg_thermal/config.cpp [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    class PF_Thermal_NVGs
+    {
+      label = "Thermal NVGs";
+      author = "OriUnder";
+      options[] =
+      {
+        "type",
+        "mode",
+        "camo",
+      //  "visor",
```
Comment (jmack) [inline PF_Gear/nvgs/nvg_thermal/config.cpp:30]: spacing

File: PF_Gear/nvgs/nvg_thermal/config.cpp [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    };
+    class PF_Gear_Nvg_Thermal_Gungnir_Orange : PF_Gear_Nvg_Thermal_Gungnir_White
+    {
+      mode = "orange";
+    };
+  };
+};
+
+class CfgWeapons
+{
```
Comment (jmack) [inline PF_Gear/nvgs/nvg_thermal/config.cpp:165]: Add some comments for each section so we can more easily tell them apart (implant, marksman, etc)

File: PF_Gear/nvgs/nvg_thermal/config.cpp [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+  class PF_Gear_Nvg_Thermal_Gungnir_White : OPTRE_NVGT_C
+  {
+    dlc = "PF_Plt";
+    author = "OriUnder";
+    displayName = "[PF] Thermal Gungnir Blastshield";
+    visionMode[] =
+    {
+    "Normal",
+    "NVG",
+    "TI",
```
Comment (jmack) [inline PF_Gear/nvgs/nvg_thermal/config.cpp:231]: spacing is off on these

**Review by jmack — CHANGES_REQUESTED** [review]

File: PF_Gear/nvgs/nvg_thermal/config.cpp (hunk 3 above)
Comment (OriUnder) [reply PF_Gear/nvgs/nvg_thermal/config.cpp]: Fixed in 
https://github.com/jmack/PF_Platoon/pull/5/commits/62fb0b15ef7dbf71a5e5f25fedd82f84c9afee5f

File: PF_Gear/nvgs/nvg_thermal/config.cpp (hunk 2 above)
Comment (OriUnder) [reply PF_Gear/nvgs/nvg_thermal/config.cpp:165]: Fixed in
https://github.com/jmack/PF_Platoon/pull/5/commits/62fb0b15ef7dbf71a5e5f25fedd82f84c9afee5f

**Review by jmack — APPROVED** [review]

**Pull request merged**